ledcontroller/__init__.py

```diff
@@ -440,3 +453,26 @@ def batch_run(self, *commands):
                 args = command[1:]
                 cmd(*args)
         self.repeat_commands = original_retries
+
+
+
+def rgb_to_hue(red, green, blue):
```

Comment:
really? The code where I copied is MIT license and it's quite small snippet. Why do you want a full attribution for one function?